scripts/viz/tsne_viz.R

```diff
@@ -1,25 +1,47 @@
 # Pan-Cancer Variational Autoencoder
 # Gregory Way 2017
 #
-# Visualize t-SNE plots of VAE and raw RNAseq data features
+# Visualize t-SNE plots of VAE, ADAGE, and raw RNAseq data features
 
 library(dplyr)
 library(ggplot2)
 
+plot_tsne <- function(tsne_df) {
+  # Function to output tsne visualizations colored by cancer type for the given
+  # input data (compressed by either Tybalt or ADAGE or raw RNAseq data)
+  p <- ggplot(tsne_df, aes(x = `1`, y = `2`, color = acronym)) +
+    geom_point(size = 0.001) +
+    scale_colour_manual(limits = tcga_colors$`Study Abbreviation`,
```

Comment:
I was personally a bit confused having the function defined before tcga_colors but I think that may be a personal style choice.

Reply:
you're right - not a style choice but a departure from proper R. I will fix!